### DISCUSSION TITLE
How to record a track with Organic Maps?

### DISCUSSION THREAD

**Discussion by tps800** [organicmaps/organicmaps#4370] [Q&A] [answered]
I am used to use OsmAnd and this app is capable to record a track you're hiking. I am using this feature most of the time and I'd like to know if this is possible with Organic Maps too?

Can I record a track while hiking and have it shown live within the map?
How could it be done if it is not build in?

**Comment by biodranik** [comment, marked as answer]
Please check existing issues. This feature is planned.

**Reply by avengine** [reply]
please add this feature asap, i am using this in alltrails all the time, very helpful, especially it can include the photos in the path too.

**Comment by foss-** [comment]
Open Feature Request: https://github.com/organicmaps/organicmaps/issues/613
